test-container/src/main/java/io/strimzi/utils/WaitException.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.utils;
+
```

Comment:
A line of Javadoc please, to explain what a WaitException is supposed to be conceptually.